subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/testsuites/TestSuitesIntegrationTest.groovy

```diff
@@ -472,8 +473,11 @@ class TestSuitesIntegrationTest extends AbstractIntegrationSpec {
             }
         """
 
+        executer.expectDocumentedDeprecationWarning("Accessing test options prior to setting test framework has been deprecated. This is scheduled to be removed in Gradle 8.0.")
```

Comment:
This is expected twice.

Reply:
What is the benefit of having twice the same deprecation warning printed?